### ISSUE TITLE
First tab-stopp is off by one to the left, resulting in default unix files not being aligned nicely

### ISSUE THREAD

**Issue by monkeydom** [sharkdp/bat#184]
E.g.:

```
dom@dreizehn:~$ cat /etc/hosts
##
# Host Database
#
# localhost is used to configure the loopback interface
# when the system is booting.  Do not change this entry.
##
127.0.0.1	localhost
255.255.255.255	broadcasthost
::1             localhost
dom@dreizehn:~$ bat /etc/hosts
───────┬────────────────────────────────────────────────────────────────────────────────────────────
       │ File: /etc/hosts
───────┼────────────────────────────────────────────────────────────────────────────────────────────
   1   │ ##
   2   │ # Host Database
   3   │ #
   4   │ # localhost is used to configure the loopback interface
   5   │ # when the system is booting.  Do not change this entry.
   6   │ ##
   7   │ 127.0.0.1      localhost
   8   │ 255.255.255.255        broadcasthost
   9   │ ::1             localhost
───────┴────────────────────────────────────────────────────────────────────────────────────────────
```

**Comment by sharkdp** [comment]
Thank you for reporting this. This is related to to #166.

**Comment by sharkdp** [comment]
Fix released in [v0.7.0](https://github.com/sharkdp/bat/releases/tag/v0.7.0).
